docs/sources/tempo/metrics-generator/service_graphs/estimate-cardinality.md

```diff
@@ -19,30 +19,46 @@ The amount of edges depends on the number of nodes in the system and the directi
 Let’s call this amount hops. Every hop will be a unique combination of client + server labels.
 
 For example:
+
 - A system with 3 nodes `(A, B, C)` of which A only calls B and B only calls C will have 2 hops `(A → B, B → C)`
 - A system with 3 nodes `(A, B, C)` that call each other (i.e., all bidirectional link) will have 6 hops `(A → B, B → A, B → C, C → B, A → C, C → A)`
 
 We can’t calculate the amount of hops automatically based upon the nodes,
 but it should be a value between `#services - 1` and `#services!`.
 
 If we know the amount of hops in a system, we can calculate the cardinality of the generated
-[service graphs]({{< relref "../service_graphs" >}}):
+[service graphs]({{< relref "../service_graphs" >}}) (assuming `#hb` is the number of histogram buckets):
 
 ```
   traces_service_graph_request_total: #hops
   traces_service_graph_request_failed_total: #hops
-  traces_service_graph_request_server_seconds: 3 buckets * #hops
-  traces_service_graph_request_client_seconds: 3 buckets * #hops
+  traces_service_graph_request_server_seconds: #hb * #hops
+  traces_service_graph_request_client_seconds: #hb * #hops
   traces_service_graph_unpaired_spans_total: #services (absolute worst case)
   traces_service_graph_dropped_spans_total: #services (absolute worst case)
 ```
 
 Finally, we get the following cardinality estimation:
 
 ```
-  Sum: 8 * #hops + 2 * #services
+  Sum: [([2 * #hb] + 2) * #hops] + [2 * #services]
+```
+
+{{% admonition type="note" %}}
+If `enable_messaging_system_latency_histogram` configuration is set to `true`, another histogram will be produced:
```

Comment:
We try to use present, when possible.  More information in the [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#write-in-present-tense).
```suggestion
If `enable_messaging_system_latency_histogram` configuration is set to `true`, another histogram is produced:
```